tensorflow_addons/image/README.md

```diff
@@ -13,6 +13,7 @@
 | distort_image_ops | random_hsv_in_yiq |  |
 | transform_ops | transform |  | 
 | transform_ops | angles_to_projective_transforms | | 
+| transform_ops | rotate | | 
```

Comment:
nitto, in alphabetical order?

Reply:
Done, but this is already way out of alphabetical order: `transform, angles..., rotate, atrices` [sic].  I've also fixed a typo.